spec/rails_helper.rb

```diff
@@ -539,36 +528,26 @@ class SpecTimeoutError < StandardError
     # failed system tests a lot trickier.
     if ENV["SELENIUM_DISABLE_VERBOSE_JS_LOGS"].blank?
       if example.exception
-        skip_js_errors = false
-
-        if example.exception.kind_of?(RSpec::Core::MultipleExceptionError)
-          lines << "~~~~~~~ SYSTEM TEST ERRORS ~~~~~~~"
-          example.exception.all_exceptions.each { |ex| lines << ex.message }
-          lines << "~~~~~ END SYSTEM TEST ERRORS ~~~~~"
-
-          skip_js_errors = true
-        end
```

Comment:
No clue why this existed cause I have never seen this working at all before.

Reply:
I added it here https://github.com/discourse/discourse/pull/16311/commits/156b41651529bbf6e302aa38e2aad5857c16d604 in the initial system spec PR...but it's not clear why now :laughing: